src/components/OutboundLink.js

```diff
@@ -0,0 +1,34 @@
+var React = require('react');
+var assign = require('react/lib/Object.assign');
+
+var OutboundLink = React.createClass({
+  displayName: 'OutboundLink',
+  statics: {
+    trackLink: function() {
+      console.warn("ga tracking not enabled");
+    }
+  },
+  handleClick: function (e) {
+    e.preventDefault();
+    var props = this.props;
+    OutboundLink.trackLink(props.args, function () {
```

Comment:
strongly recommend renaming "args" to something that explains what it really is, since legibility > short code (minification will do all the shortening we need =D)